services/map.spec.lua

```diff
@@ -1,22 +1,15 @@
 _G.love = require 'lib/love'
 local Map = require 'services/map'
 local Textures = require 'services/textures'
+local mapList = require 'maps/mapList'
 
 describe('Map Service', function()
     Textures.load()
 
-    it('loads & draws "happy"', function()
-        Map.load("happy")
-        Map.draw()
-    end)
-
-    it('loads & draws "no_planets"', function()
-        Map.load("no_planets")
-        Map.draw()
-    end)
-
-    it('loads & draws "test"', function()
-        Map.load("test")
-        Map.draw()
-    end)
+    for _, map in ipairs(mapList) do
+        it('loads & draws ' .. map.displayName , function()
+            Map.load(map.filename)
+            Map.draw()
```

Comment:
Should we go ahead and unload it. I know it wont do much. But to test unloading.

Reply:
I honestly don't know. This is @jaythomas 's domain. I just extended his unit test to actually cover all the maps.